tests/unit/datepicker/core.js

```diff
@@ -540,4 +540,83 @@ QUnit.test( "mouse", function( assert ) {
 		"Mouse click inline - next" );
 } );
 
+QUnit.test( "position", function( assert ) {
+	assert.expect( 4 );
+
+	var $container = $( "#qunit-fixture" ),
+		$modal = $container.children('div'),
```

Comment:
That's a bit fragile to depend on it always being a single div. If you need a reference to this element, can you maybe start from the `inp` input and go to the parent?